diff --git a/core/modules/file/src/FileViewsData.php b/core/modules/file/src/FileViewsData.php
index aad5ac23a4..1c51b200dd 100644
--- a/core/modules/file/src/FileViewsData.php
+++ b/core/modules/file/src/FileViewsData.php
@@ -121,6 +121,7 @@ public function getViewsData() {
       'skip base' => ['node_field_data', 'node_field_revision', 'users_field_data', 'comment_field_data', 'taxonomy_term_field_data'],
       'real field' => 'id',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('Content'),
         'label' => $this->t('Content'),
         'base' => 'node_field_data',
@@ -137,6 +138,7 @@ public function getViewsData() {
       'skip base' => ['file_managed', 'users_field_data', 'comment_field_data', 'taxonomy_term_field_data'],
       'real field' => 'fid',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('File'),
         'label' => $this->t('File'),
         'base' => 'file_managed',
@@ -154,6 +156,7 @@ public function getViewsData() {
       'skip base' => ['node_field_data', 'node_field_revision', 'users_field_data', 'comment_field_data', 'taxonomy_term_field_data'],
       'real field' => 'id',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('User'),
         'label' => $this->t('User'),
         'base' => 'users',
@@ -170,6 +173,7 @@ public function getViewsData() {
       'skip base' => ['file_managed', 'node_field_data', 'node_field_revision', 'comment_field_data', 'taxonomy_term_field_data'],
       'real field' => 'fid',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('File'),
         'label' => $this->t('File'),
         'base' => 'file_managed',
@@ -187,6 +191,7 @@ public function getViewsData() {
       'skip base' => ['node_field_data', 'node_field_revision', 'users_field_data', 'comment_field_data', 'taxonomy_term_field_data'],
       'real field' => 'id',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('Comment'),
         'label' => $this->t('Comment'),
         'base' => 'comment_field_data',
@@ -203,6 +208,7 @@ public function getViewsData() {
       'skip base' => ['file_managed', 'node_field_data', 'node_field_revision', 'users_field_data', 'taxonomy_term_field_data'],
       'real field' => 'fid',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('File'),
         'label' => $this->t('File'),
         'base' => 'file_managed',
@@ -220,6 +226,7 @@ public function getViewsData() {
       'skip base' => ['node_field_data', 'node_field_revision', 'users_field_data', 'comment_field_data', 'taxonomy_term_field_data'],
       'real field' => 'id',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('Taxonomy Term'),
         'label' => $this->t('Taxonomy Term'),
         'base' => 'taxonomy_term_data',
@@ -236,6 +243,7 @@ public function getViewsData() {
       'skip base' => ['file_managed', 'node_field_data', 'node_field_revision', 'users_field_data', 'comment_field_data'],
       'real field' => 'fid',
       'relationship' => [
+        'id' => 'standard',
         'title' => $this->t('File'),
         'label' => $this->t('File'),
         'base' => 'file_managed',
